site/static/docs/5.3/assets/img/boosted-warning.svg

```diff
@@ -0,0 +1,4 @@
+<svg xmlns='http://www.w3.org/2000/svg' fill="currentColor" viewBox="0 0 1000 1000">
```

Comment:
We can probably drop the `fill="currentColor"` part as it won't be used:

```suggestion
<svg xmlns='http://www.w3.org/2000/svg' viewBox="0 0 1000 1000">
```

Reply:
You're right, it is not consistent... I tried with fill="currentColor", it doesn't work in this context 'external svg image), so I remove it and keep fill="#fc0"